### DISCUSSION TITLE
RPI zero W with hifiberry miniamp

### DISCUSSION THREAD

**Discussion by githans1** [rern/rAudio#1205] [General]
Does anybody have any experience with the hifiberry miniamp? I am experimenting with it in combination with the rpi zero. Unfortunately the miniamp is not listed in the device list. 

I have tried "amp", but result is "no device" in the settings.
Also tried "dac", the result gives "no mixer" in the settings.

All in all, the combinations run acc to the interface but zero sound.

Any suggestions?

**Comment by one-step-behind** [comment]
Hi @githans1. I'm running a Rpi 3B with a HiFiBerry MiniAmp - which is btw great for running a battery powered Musicbox. You have to set the audio interface to "HiFiBerry DAC" then restart! Afterwards you should have the option in volume control to set it to "MPD software" in player settings.

**Reply by githans1** [reply]
Hi @one-step-behind 

I managed to get it working this way. Thank you for your answer.

Did you also get the equaliser working? The audio interface says "no mixer", and I seem not to be able to set and save the equaliser. After saving a new setting it jumps back to flat. I would like to have less trebble and more bass. 

Update: It turns out that the equalizer is actually working. It is just very slow when saving the new setting. It can take up to several minutes.

**Reply by one-step-behind** [reply]
Yes, equalizer is working for me. If you change the EQ it will be saved immediately to a JSON file (in earlier versions of rAudio it was a different file and you had to save it via a dedicated save button). Due to the fact of immediate saving it could be that it takes time to save your EQ settings if your (wireless) connection to the RasPi is slow.

_Sorry for not getting back earlier. I had to dig into the equalizer settings again._

**Comment by githans1** [comment]
My RasPi Zero with miniamp is slow in general. Eg changing volume takes 
generally a few seconds to react. Also booting takes a few minutes. 
Other functions like eg changing song react instantly. Doesn't really 
bother me btw.
